src/Dropdown.jsx

```diff
@@ -1,3 +1,6 @@
+/* eslint react/no-find-dom-node: 0 */
+// https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-find-dom-node.md
```

Comment:
> Facebook will eventually deprecate findDOMNode as it blocks certain improvements in React in the future.

Might be worth keeping track of this discussion at https://github.com/yannickcr/eslint-plugin-react/issues/678